llvm/test/CodeGen/ARM/fnmscs.ll

```diff
@@ -13,7 +13,7 @@
 ; RUN: llc -mtriple=arm-eabi -mcpu=cortex-a8 -regalloc=basic %s -o - \
 ; RUN:  | FileCheck %s -check-prefix=A8
 
-; RUN: llc -mtriple=arm-eabi -mcpu=cortex-a8 --enable-unsafe-fp-math %s -o - \
+; RUN: llc -mtriple=arm-eabi -mcpu=cortex-a8 --enable-unsafe-fp-math  %s -o - \
```

Comment:
"denormal-fp-math=preserve-sign" is conflict with fsub dag combiner, we should have instruction level denormal flags, like fast math flags, see #30633.